Course_Part_07/src/assignment29/SavingsAccount.java

```diff
@@ -0,0 +1,27 @@
+package assignment29;
+
+public class SavingsAccount extends Account {
+    int interest;  // gekozen voor default access modifier (ook methodes), alleen beschikbaar voor deze package. De subclass is nu niet te gebruiken vanuit een andere package.
+
+    public SavingsAccount() {
+        this.name = name;
+        this.balance = 0;
+        this.interest = 5;
+    }
+
+
+    @Override
+    void withDraw(int amount) {
+        if (balance - amount < 0) {
+            System.out.println (String.format ("Your balance is negative, it's not possible to withdraw € %d", amount));  // kan ik hier ook printf gebruiken?
```

Comment:
Ja kan idd, namelijk:

```
System.out.printf("Your balance is negative, it's not possible to withdraw € %d%n", amount);
```

printf kan ook werken met de `%s, %d` notatie. `%n` betekent hier newline. 

In de cursus gebruiken we println icm String.format, dat is idd minder efficiënt dan printf, maar willen daar niet te veel tijd aan besteden. Voel je vrij om de suggestie van Intellij over te nemen of zelf zou uit te werken.